### PR TITLE
Update Avalonia About Window like requested in PR #6267

### PR DESCRIPTION
**Pull request by sunshineinabox**
Update about window to have Logo and "Ryujinx" in center and adjacent as requested in PR #6267.

---

Before:
image

After:
image

### CODE REVIEW

File: src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml [hunk 1]
```diff
@@ -44,31 +44,36 @@
...
                         <TextBlock
-                            HorizontalAlignment="Center"
-                            VerticalAlignment="Center"
                             FontSize="28"
                             FontWeight="Bold"
                             Text="Ryujinx"
-                            TextAlignment="Center"
-                            Width="110" />
-                        <TextBlock
+                            TextAlignment="Left"
```
Comment (IsaacMarovitz) [inline src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml:65]: `TextAlignment=Start` should be used instead of Left to avoid issues in RTL

**Review by marysaka — APPROVED** [review]

File: src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml [hunk 2]
```diff
@@ -44,31 +44,36 @@
                         <ColumnDefinition Width="*" />
                         <ColumnDefinition Width="Auto" />
                     </Grid.ColumnDefinitions>
+                    <StackPanel
+                        Grid.Column="1"
+                        Orientation="Horizontal"
+                        HorizontalAlignment="Center"
+                        Spacing="10">
```
Comment (IsaacMarovitz) [inline src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml:51]: Children of this StackPanel should be indented

File: src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml [hunk 3]
```diff
@@ -44,31 +44,36 @@
...
                     <Image
-                        Grid.Column="0"
                         Height="80"
-                        Source="resm:Ryujinx.Ui.Common.Resources.Logo_Ryujinx.png?assembly=Ryujinx.Ui.Common" />
+                        Source="resm:Ryujinx.Ui.Common.Resources.Logo_Ryujinx.png?assembly=Ryujinx.Ui.Common"
+                        HorizontalAlignment="Center"
+                        IsHitTestVisible="True"/>
```
Comment (IsaacMarovitz) [inline src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml:56]:
```suggestion
                        IsHitTestVisible="True" />
```

File: src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml [hunk 4]
```diff
@@ -44,31 +44,36 @@
...
                             FontWeight="Bold"
                             Text="Ryujinx"
-                            TextAlignment="Center"
-                            Width="110" />
-                        <TextBlock
+                            TextAlignment="Start"
+                            Width="110"
                             HorizontalAlignment="Center"
-                            VerticalAlignment="Center"
+                            VerticalAlignment="Center"/>
```
Comment (IsaacMarovitz) [inline src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml:68]:
```suggestion
                            VerticalAlignment="Center" />
```

File: src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml [hunk 5]
```diff
@@ -44,31 +44,36 @@
...
+                        <TextBlock
                             FontSize="11"
                             Text="(REE-YOU-JINX)"
-                            TextAlignment="Center"
-                            Width="110" />
-                    </WrapPanel>
+                            TextAlignment="Start"
+                            Width="110"
+                            HorizontalAlignment="Center"
+                            VerticalAlignment="Center"/>
```
Comment (IsaacMarovitz) [inline src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml:75]:
```suggestion
                            VerticalAlignment="Center" />
```

File: src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml [hunk 6]
```diff
@@ -44,31 +44,36 @@
...
+                    </WrapPanel></StackPanel>
```
Comment (IsaacMarovitz) [inline src/Ryujinx.Ava/UI/Windows/AboutWindow.axaml:76]: Indentation

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

**Review by IsaacMarovitz — APPROVED** [review]

**Review by AcK77 — APPROVED** [review]

**Pull request merged**